addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/handler/RFXComBridgeHandler.java

```diff
@@ -0,0 +1,430 @@
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.binding.BaseBridgeHandler;
+import org.eclipse.smarthome.core.types.Command;
+import org.eclipse.smarthome.core.types.RefreshType;
+import org.openhab.binding.rfxcom.RFXComBindingConstants;
+import org.openhab.binding.rfxcom.internal.DeviceMessageListener;
+import org.openhab.binding.rfxcom.internal.RFXComException;
+import org.openhab.binding.rfxcom.internal.config.RFXComConfiguration;
+import org.openhab.binding.rfxcom.internal.connector.RFXComConnectorInterface;
+import org.openhab.binding.rfxcom.internal.connector.RFXComEventListener;
+import org.openhab.binding.rfxcom.internal.connector.RFXComJD2XXConnector;
+import org.openhab.binding.rfxcom.internal.connector.RFXComSerialConnector;
+import org.openhab.binding.rfxcom.internal.messages.RFXComInterfaceMessage;
+import org.openhab.binding.rfxcom.internal.messages.RFXComInterfaceMessage.Commands;
+import org.openhab.binding.rfxcom.internal.messages.RFXComInterfaceMessage.TransceiverType;
+import org.openhab.binding.rfxcom.internal.messages.RFXComBaseMessage;
+import org.openhab.binding.rfxcom.internal.messages.RFXComMessageFactory;
+import org.openhab.binding.rfxcom.internal.messages.RFXComMessage;
+import org.openhab.binding.rfxcom.internal.messages.RFXComTransmitterMessage;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * {@link RFXComBridgeHandler} is the handler for a RFXCOM transceivers. All
+ * {@link RFXComHandler}s use the {@link RFXComBridgeHandler} to execute the
+ * actual commands.
+ * 
+ * @author Pauli Anttila - Initial contribution
+ */
+public class RFXComBridgeHandler extends BaseBridgeHandler {
+
+	private Logger logger = LoggerFactory.getLogger(RFXComBridgeHandler.class);
+
+	RFXComConnectorInterface connector = null;
+	private MessageLister eventLister = new MessageLister();
+
+	private List<DeviceMessageListener> deviceStatusListeners = new CopyOnWriteArrayList<>();
+
+	private static final int timeout = 5000;
+	private static byte seqNbr = 0;
+	private static RFXComTransmitterMessage responseMessage = null;
+	private Object notifierObject = new Object();
+	private RFXComConfiguration configuration = null;
+
+	public RFXComBridgeHandler(Bridge br) {
+		super(br);
+	}
+
+	@Override
+	public void handleCommand(ChannelUID channelUID, Command command) {
+		if (command instanceof RefreshType) {
+			logger.debug("Refresh command received.");
+
+		} else {
+			logger.warn("No bridge commands defined.");
+		}
+	}
+
+	@Override
+	public void dispose() {
+		logger.debug("Handler disposed.");
+
+		if (connector != null) {
+			connector.removeEventListener(eventLister);
+			connector.disconnect();
+		}
+
+		super.dispose();
+	}
+
+	@Override
+	public void initialize() {
+		logger.debug("Initializing RFXCOM bridge handler.");
+
+		configuration = getConfigAs(RFXComConfiguration.class);
+		connect();
```

Comment:
As initialize() should return fast, you better call connect() from within a Job.
What happens if the connection is lost (e.g. because the USB dongle is removed?) Will it automatically reconnect once the dongle is back? You might want to do a regular check and call connect() again if the connection was lost.